src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -387,6 +387,7 @@ function Test-ApplicationGatewayCRUD2
 		$gipconfig = New-AzureRmApplicationGatewayIPConfiguration -Name $gipconfigname -Subnet $gwSubnet
 
 		# frontend part
+		[SuppressMessage("Microsoft.Security", "CS002:SecretInNextLine")]
```

Comment:
supposed to be under the comment
e.g. [like this](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayTests.ps1#L323)